### ISSUE TITLE
Bug: import from another application

### ISSUE THREAD

**Issue by Altonss** [TheLastProject/Catima#660]
When importing from another application and clicking "OK" on the menu that opens, my WhatsApp contact list opens! This is a weird bug ^

**Comment by TheLastProject** [comment]
Well, the whole point of "import from another application" is to open another application. No clue why WhatsApp decides to respond to that intent though.

**Comment by Altonss** [comment]
It is maybe due to the fact that I don't heve these apps on my phone (like Stocard...). Are you able to reproduce it with an emulator?

**Comment by TheLastProject** [comment]
Oh I can reproduce it, but I am not sure if it's worth trying to work around if the feature otherwise works fine for opening another app.

**Comment by Altonss** [comment]
Yes it's just a minor bug. If someday someone wants to work on it, it has at least been documented :)

**Comment by Altonss** [comment]
I retried this import feature, and even with LoyaltyCardKeychain installed, when clicking on import from this app WhatsApp opens up.

**Comment by ZyxerZyxani** [comment]
Have you checked your settings? If perhaps you one time accidentally set Whatsapp as default for something else as well? 
Settings > apps and notifications > Advanced > Default apps.
Can also check opening links in that same menu (in the default apps)

**Comment by Altonss** [comment]
> Have you checked your settings? If perhaps you one time accidentally set Whatsapp as default for something else as well? Settings > apps and notifications > Advanced > Default apps. Can also check opening links in that same menu (in the default apps)

I don't have this setting on my old android device. But I just did a reset of all the default apps and this did not help!

**Comment by TheLastProject** [comment]
So... the previous dev added this and through the years:
- I've not used it a single time
- Nobody ever told me they used it
- 3 people said it is broken

Given I'm not quite sure how it is supposed to work, maybe the easiest fix is to only allow importing from the filesystem and just removing this button? 😅
